homeassistant/components/iqvia/sensor.py

```diff
@@ -104,7 +104,7 @@ class ForecastSensor(IQVIAEntity):
     @callback
     def update_from_latest_data(self):
         """Update the sensor."""
-        if not self.coordinator.data:
+        if not self.coordinator.last_update_success:
```

Comment:
```suggestion
        if not self.available:
```
You can use this instead.
https://github.com/home-assistant/core/blob/b1fc05413abf61449e2d12e4162332352b84c67c/homeassistant/helpers/update_coordinator.py#L310-L313